### PR TITLE
Add trend card pattern

### PR DESCRIPTION
**Pull request by serenamarie125**
Add the pattern as it currently exists on patternfly.org

### CODE REVIEW

**Comment by andresgalante** [comment]
Awesome! thanks a lot @serenamarie125 😄 

@LHinson I'd merge this one.

File: pattern-library/cards/trend-card/design/design.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      -- Left of the sparkline
+
+  1. **Total Value** (optional)**:**
+    - If needed, provide total value next to the actual value to help provide context.
+
+  1. **Unit of Measurement:**
+    - The label for unit of measurement is shown after the the value.
+
+  1. **Sparkline:**
+    - See the [Sparkline pattern](../pattern-library/data-visualization/sparkline/) for more details.
```
Comment (LHinson) [inline pattern-library/cards/trend-card/design/design.md]: Link does not work.

File: pattern-library/cards/trend-card/design/design.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  1. **Tooltip:**
+    - Use a tooltip on hover to display additional information about exact points on the sparkline (e.g. values and/or percentages). A vertical line and marker helps to accentuate the hover point.
+
+  1. **Time Frame** (optional)**:**
+    - The time frame is indicated by placing a label on one of the following areas:
+      -- Under the title
+      -- Under the sparkline
+      -- In a [time frame filter](../pattern-library/dashboard/dashboard-card/#example-overview-2)
```
Comment (LHinson) [inline pattern-library/cards/trend-card/design/design.md]: Formatting issue. "Under the title", "Under the sparkline" and "In a time frame filter" should all appear on ther own bulleted linee Also the "time frame filter" link does not work.

File: pattern-library/cards/trend-card/design/design.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - The time frame on a card is optional since there could be a single instance of the time frame label or time frame filter that applies to all cards within a single view.
+
+  1. **Actions Panel** (optional)**:**
+    - See [actions panel](../pattern-library/dashboard/dashboard-card) under the Dashboard Cards pattern for more details.
```
Comment (LHinson) [inline pattern-library/cards/trend-card/design/design.md]: Link does not work

File: pattern-library/cards/trend-card/design/overview.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Trend Card
+
+## Overview
+A trend card displays a trend of one or more metrics. The most common use case for this pattern is in a dashboard. For a single trend card, there are two different sizes of cards to choose from dependent on your application's needs.
+
+Jump to [Small Cards](../pattern-library/cards/trend-card/#example-overview-1) or [Large Cards](../pattern-library/cards/trend-card/#example-overview-2)
```
Comment (LHinson) [inline pattern-library/cards/trend-card/design/overview.md]: links are not working

**Comment by serenamarie125** [comment]
@LHinson @andresgalante I think i finally have the links working correctly :roll_eyes:!

**Comment by LHinson** [comment]
bitmoji

**Pull request merged**